packages/next/build/webpack/loaders/next-serverless-loader/api-handler.ts

```diff
@@ -41,8 +50,8 @@ export function getApiHandler(ctx: ServerlessHandlerCtx) {
       }
 
       await apiResolver(
-        req,
-        res,
+        req.req,
+        res.res,
```

Comment:
Also mentioned below, I think this is confusing so `req.originalRequest` and `res.originalResponse` might be more explicit.